packages/ember-glimmer/lib/components/link-to.js

```diff
@@ -711,6 +712,26 @@ const LinkComponent = EmberComponent.extend({
 
     let routing = get(this, '_routing');
     let queryParams = get(this, 'queryParams.values');
+
+    runInDebug(() => {
+      /*
+       * Unfortunately, to get decent error messages, we need to do this.
+       * In some future state we should be able to use a "feature flag"
+       * which allows us to strip this without needing to call it twice.
+       *
+       * if (isProdBuild()) {
```

Comment:
I believe either the truthy/falsey blocks should be flip/flopped or this conditional should be changed to `isDebugBuild()` (since we want to do the most performant thing in prod mode)...